P5/Source/Specs/eventName.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="namesdates" ident="eventName">
+  <gloss versionDate="2023-05-02" xml:lang="en">name of an event</gloss>
+  <gloss versionDate="2023-05-02" xml:lang="de">Name eines Ereignisses</gloss>
+  <desc versionDate="2023-05-02" xml:lang="en">contains a proper noun or noun phrase used to refer to an event.</desc>
+  <desc versionDate="2023-05-02" xml:lang="de">enthält einen Eigennamen in Form eines Nomens oder einer Nominalphrase, der verwendet wird, um auf ein Ereignis zu verweisen.</desc>
+  <classes>
+    <memberOf key="att.datable"/>
+    <memberOf key="att.editLike"/>
+    <memberOf key="att.global"/>
```

Comment:
Very small little copy editing thing: as discussed in F2F Paderborn, we tend to order `class/memberOf` with att.global first, and then the rest in alphabetical order.